src/adapters/simpleble-adapter.ts

```diff
@@ -53,6 +53,24 @@ export class SimplebleAdapter extends EventEmitter implements BluetoothAdapter {
     private descriptors = new Map<string, string[]>();
     private charEvents = new Map<string, (value: DataView) => void>();
 
+    getHardwareAdapters() {
+        return getAdapters().map(
+            ({ identifier, address, active }) =>
+                ({ identifier, address, active })
+        );
+    }
+
+    useHardwareAdapter(adapterIndex: number) {
```

Comment:
```suggestion
    public useHardwareAdapter(adapterIndex: number) {
```

Please make this explicitly public